src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -74,7 +74,12 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 
   <PropertyGroup>
     <DefaultValueTuplePackageVersion>4.4.0</DefaultValueTuplePackageVersion>
-    <DefaultFSharpCorePackageVersion>{{FSharpCoreShippedPackageVersion}}</DefaultFSharpCorePackageVersion>
+    <!--
+        if $(DisableFSharpCorePreviewCheck) then use existing behaviour
+        otherwise pick fsharp.core version based on $(_NETCoreSdkIsPreview)
```

Comment:
If preview, we should use version `$(FSharpCorePreviewPackageVersion)` which will look like `4.7.2-beta.*`.